### DISCUSSION TITLE
Method for tracing from rst input to generated HTML files

### DISCUSSION THREAD

**Discussion by kentj** [sphinx-doc/sphinx#12936] [Q&A]
I've been asked to work on a generic use case for translating some content from a series of rst files into an HTML version.  Is there any way to incorporate (in some fashion, comments, html element attributes, etc) some way to attribute that content back to the original rst file/line number?

Basically, leadership would like to "review" documentation in HTML format, but for collecting feedback, we will need a way to know how to apply that feedback against the original rst files.

**Comment by jdillard** [comment]
There is not (or at least not in a way that is remotely worth the effort). Authors of the document should know which lines to change, similar to how developers of websites should know what lines to change. I would recommend another way of collecting feedback that meets their needs.
